boards/arm/qemu_cortex_r5/qemu_cortex_r5.dts

```diff
@@ -6,7 +6,7 @@
  */
 
 /dts-v1/;
-#include <arm/xilinx/zynqmp.dtsi>
+#include <arm/xilinx/zynqmp_rpu.dtsi>
 
 / {
 	model = "QEMU Cortex-R5";
```

Comment:
Do we want to specify RPU in the model and compatible as well?

Reply:
For `model`, this is a "QEMU Cortex-R5" board that just happens to be using Xilinx ZynqMP, so I don't think "RPU" (Xilinx ZynqMP-specific term) belongs there. In addition, "Cortex-R5" pretty much implies "RPU." This also aligns with what `qemu_cortex_m0` and `qemu_cortex_m3` are doing.

As for `compatible`, this field has no real function (at least for now) and the common trend seems to be to specify the compatible SoC name. Since the SoC is `xlnx,zynqmp` regardless of RPU or APU, I think we should leave it as is (`#include dtsi` is what really determines which DT is used by board). Regarding the `-qemu` suffix, this probably should be removed unless we have a specific reason to differentiate qemu vs. non-qemu for zynqmp (e.g. as was done in as `qemu_cortex_m3`).